runtime/doc/develop.txt

```diff
@@ -227,7 +227,7 @@ in src/nvim/api/win_config.c like this: >
 
 
 Lua docstrings ~
-							*dev-lua-doc*
+							*dev-doc-lua*
```

Comment:
the old tag is correct, the idea is that we will eventually have more `dev-lua-xx` sections

Reply:
Flipped in pr